GitHubCard/index.js

```diff
@@ -30,6 +39,29 @@
 
 const followersArray = [];
 
+followersArray.push('tetondan');
```

Comment:
You can also initialize the array with these names like this 
`const followersArray = ['tetondan', 'dustinmyers', 'etc']`

Normally we use `.push()` when we need to add data dynamically. That is to say, we don't know what data will belong in the array when we are writing code. Perhaps, for example, we are adding elements to an array based on user input. 

But for your case, since you know all the elements ahead of time, you should initialize the array with the data.

Reply:
ok thanks for the advice :)